frontend/src/components/uam/helpers/get-rows-helper/get-users-rows.helper.ts

```diff
@@ -1,12 +1,19 @@
 import { UsersGetResponseDto } from 'common/types/types';
 import { UserTableAccessor } from 'components/uam/common/enums/enums';
+import { UsersTableType } from 'components/uam/common/types/types';
 
-const getUsersRows = (users: UsersGetResponseDto[]): UsersGetResponseDto[] => {
+import { Actions } from '../../components/users-table/components/components';
+
+const getUsersRows = (
+  users: UsersGetResponseDto[],
+  onUserDelete: (userId: string) => void,
+): UsersTableType[] => {
   return users.map((user) => ({
     [UserTableAccessor.ID]: user.id,
     [UserTableAccessor.EMAIL]: user.email,
     [UserTableAccessor.FULL_NAME]: user.fullName,
     [UserTableAccessor.CREATED_AT]: user.createdAt,
+    [UserTableAccessor.ACTIONS]: Actions(String(user.id), onUserDelete),
```

Comment:
Do not provide component as accessor value for table
```suggestion
    [UserTableAccessor.ACTIONS_PAYLOAD]: user.id,
```